opencensus/trace/from_context.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPENCENSUS_TRACE_FROM_CONTEXT_H_
+#define OPENCENSUS_TRACE_FROM_CONTEXT_H_
```

Comment:
If we want to add in the future things like "AddAnnotationToCurrentSpan" (or whatever name we pick) is this a good header name to be used for these helpers as well? What about "context_util", "trace_context"?

Reply:
SGTM